pages/assessment.tsx

```diff
@@ -5,9 +5,11 @@ import { useState } from "react";
 import data from "../data/IT2810Høst2018.json";
 import Expand from "../components/expand";
 import {
+  clearLocalStorage,
```

Comment:
Where is this being used?

Reply:
I use it sometimes to clear my localstorage for testing purposes. It gets automatically imported and I forgot to remove it.